sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -115,10 +115,10 @@ export interface CertificateIssuer extends IssuerProperties {
 }
 
 // @public
-export type CertificateKeyCurveName = string;
+export type CertificateKeyCurveName = "P-256" | "P-384" | "P-521" | "P-256K";
```

Comment:
This API diff results from the new approach to extensible enums in the codegen. Currently these enums are re-exported from the generated code directly. But we can handroll these types instead if we want to keep the API the same.